specification/apimanagement/resource-manager/Microsoft.ApiManagement/stable/2022-08-01/apimanagement.json

```diff
@@ -0,0 +1,835 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "ApiManagementClient",
+    "description": "Use these REST APIs for performing operations on entities like API, Product, and Subscription associated with your Azure API Management deployment.",
+    "version": "2022-08-01"
```

Comment:
>2022-08-01" [](http://example.com/codeflow?start=16&length=11)

Please add the 2022-04-01-preview version files as is in the first commit and then make all changes from the subsequent ones

Reply:
That is what we did
https://github.com/Azure/azure-rest-api-specs/pull/21169/commits/a78db08f23f2f0694236abe8aa8eaaf9472cc3dc